libraries/rush-lib/src/logic/PackageJsonUpdater.ts

```diff
@@ -265,6 +310,90 @@ export class PackageJsonUpdater {
     }
   }
 
+  /**
+   * Remove a dependency from a particular project. The core business logic for "rush remove".
+   */
+  public async doRushRemoveAsync(options: IPackageJsonUpdaterRushRemoveOptions): Promise<void> {
```

Comment:
There's a fair amount of code duplicated between this and `doRushAddAsync`. Can these be DRYed up?

Reply:
done